backend/server.js

```diff
@@ -284,17 +289,17 @@ function main() {
     //------------------------------------------------------------------------------
     // json poop
     //------------------------------------------------------------------------------
-    importParameters("recordings/test_parameters.json")
+    importParameters("../recordings/test_parameters.json")
```

Comment:
@pbucci I think ../recordings works only if running server.js separately. But since we are using npm start, I will just change it to ./recording (This won't break npm start, but it might be safer to check again with a servo before merging. I will tell you once it's checked! ).